corehq/apps/export/views/utils.py

```diff
@@ -415,3 +415,5 @@ def clean_odata_columns(export_instance):
             # truncate labels for PowerBI and Tableau limits
             if len(column.label) >= 255:
                 column.label = column.label[:255]
+            if column.label in ['formid'] and column.is_deleted:
```

Comment:
I'm assuming we now have a mechanism in place to prevent `formid` from being used a question id, so we won't run into a case where duplicate `formid` columns exist that are _not_ deleted?

Reply:
yes I believe so. this was discussed in the ticket and in the description ^_^

Reply:
I didn't phrase my question well. I'm imagining a case where a duplicate `formid` question exists, and the ExportColumn's `is_deleted` field is somehow still false? Is that even possible/something we need to worry about in old app builds?